packages/react-codemirror/playwright-ct.config.ts

```diff
@@ -42,16 +42,7 @@ export default defineConfig({
   },
 
   // Glob patterns or regular expressions that match test files.
-  testMatch: '*.spec.tsx',
+  testMatch: '**/*.spec.tsx',
 
-  projects: [
-    {
-      name: 'chromium',
-      use: { ...devices['Desktop Chrome'] },
-    },
-    {
-      name: 'firefox',
-      use: { ...devices['Desktop Firefox'] },
-    },
-  ],
+  projects: [{ name: 'chromium', use: { ...devices['Desktop Chrome'] } }],
```

Comment:
i think it makes sense to create a test suite dedicated for webkit + safari at a later stage and avoid the flakyness from firefox now. also happy to add firefox back if you prefer

Reply:
I think we have firefox here because at some point we broke something in the semantic analysis that was firefox specific. To avoid regressions I think it'd be nice to still keep it in the CI?

Reply:
We didn't fix the semantic analysis in firefox, it's still not working. I'll add it back :+1:

Reply:
> We didn't fix the semantic analysis in firefox, it's still not working.

Not completely you are right. But it does work in my Firefox, when it wasn't before. We definitely fixed something that wasn't working at some point that had to do with us setting the schema eagerly.